cellxgene_schema_cli/tests/test_ontology.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+
+from cellxgene_schema import ontology
+
+class TestGeneChecker(unittest.TestCase):
+    def setUp(self):
+        self.valid_species = ["Homo sapiens", "Mus musculus"]
+        self.invalid_species = ["Caenorhabditis elegans"]
+
+        self.valid_genes = {
+            "Homo sapiens": {
+                "ENSG00000141510": "TP53"
+            },
+            "Mus musculus": {
+                "ENSMUSG00000059552": "Trp53"
+            }
+        }
+
+        self.invalid_genes = {
+            "Homo sapiens": ["ENSMUSG00000059552", "GENE"],
+            "Mus musculus": ["ENSG00000141510", "GENE"]
+
+        }
+
+
+    def test_species_validity(self):
+        for species in self.valid_species:
+            self.assertIsInstance(ontology.geneChecker(species), ontology.geneChecker)
+        for species in self.invalid_species:
+            with self.assertRaises(ValueError):
+                ontology.geneChecker(species)
+
+    def test_valid_genes(self):
+        for species in self.valid_genes:
+            geneChecker = ontology.geneChecker(species)
+            for gene_id in self.valid_genes[species]:
+                gene_label = self.valid_genes[species][gene_id]
+
+                self.assertTrue(geneChecker.is_valid_id(gene_id))
+                self.assertEqual(geneChecker.get_symbol(gene_id), gene_label)
+
+    def test_invalid_genes(self):
+        for species in self.invalid_genes:
+            geneChecker = ontology.geneChecker(species)
+            for gene_id in self.invalid_genes[species]:
+
+                self.assertFalse(geneChecker.is_valid_id(gene_id))
+                with self.assertRaises(ValueError):
+                    geneChecker.get_symbol(gene_id)
+
+
+
+class TestOntologyChecker(unittest.TestCase):
+    def setUp(self):
+        self.ontologyChecker = ontology.ontologyChecker()
```

Comment:
Unless you want a new instance of the ontologyChecker for each test you may want to use 
```
@classmethod
    def setUpClass(cls):
```

Reply:
Corrected in https://github.com/chanzuckerberg/single-cell-curation/pull/99/commits/811f16ac4dea3f95325c08d1f4639a57f3e698d1